ftp.go

```diff
@@ -320,7 +314,7 @@ func parseListLine(line string) (*Entry, error) {
 					e.Type = EntryTypeFile
 				}
 			case "size":
-				e.Size, _ = strconv.ParseUint(value, 0, 64)
+				e.setSize(value)
```

Comment:
We have to use `strconv.ParseUint(value, 0, 64)`:
https://play.golang.org/p/p4L2Y9F4cA
I updated my pull request.